Directory.Build.props

```diff
@@ -6,6 +6,7 @@
   <PropertyGroup>
     <Copyright>$(CopyrightNetFoundation)</Copyright>
     <PackageLicenseExpression>MIT</PackageLicenseExpression>
+    <MainLibraryPath>$(MSBuildThisFileDirectory)src/System.Device.Gpio/</MainLibraryPath>
```

Comment:
why not define path directly to csproj?

Reply:
We don't want to keep a copy around on every single .csproj we have, since in case we ever decide to restructure the repo, it would mean we would have to edit this property everywhere. This way, we only have it declared in a central place, so modifying it in the future is trivial.